packages/application-shell/src/components/project-switcher/project-switcher.tsx

```diff
@@ -32,6 +33,8 @@ type CustomValueContainerProps = ValueContainerProps & {
   projectCount: number;
 };
 
+const PROJECT_SWITCHER_LABEL_ID = 'project-switcher-label';
```

Comment:
We have a constant field since there is only one instance of this component in the page.